docs/releases/versions/version-v12.0.0.md

```diff
@@ -0,0 +1,117 @@
+# InvenioRDM v12.0
+
+_2024-07-31_
+
+_Long-term support (LTS) release_
+
+We're happy to announce the release of InvenioRDM v12.0, the new much anticipated long-term support release! Version 12 will be maintained until at least 6 months following the next long-term support release. This also means that in February 2025, we will retire v9.1, the previous LTS. Visit our [maintenance policy page](../maintenance-policy.md) to learn more.
+
+## Try it
+
+- [Demo site](https://inveniordm.web.cern.ch)
+
+- [Installation instructions](https://inveniordm.docs.cern.ch/install/)
+
+## What's new?
+
+A lot is new since the last v11 release. This new version is brimming with new functionality and improvements. We highlight the big changes here and at the end of this page we collate a list of the miscellaneous other improvements.
+
+### Powerful and reliable record access
+
+![Share button](../../features/features-walk-through/img/access_request_share_button.png)
+
+[Giving and requesting access](../../features/features-walk-through/access_requests.md) to records have seen a complete overhaul in this release. Record owners or curators can share them directly with other users or with groups, as well as control whether, who and how access can be demanded.
+
+![Share modal](../../features/features-walk-through/img/access_requests_tab.png)
+
+
+This provides flexible, yet reliable access control to the records of your instance and can even be done in bulk.
+
+### Notifications
+
+![Notifications Setting Menu Entry](../../features/features-walk-through/img/notifications/menu.png "Notification Settings Menu Entry")
+
+[A notification system](../../features/features-walk-through/notifications.md) has been introduced. Users can now receive email notifications when they are involved in certain activities. For instance, invitations to a community will send an email to the invited party and notify the inviter of the invitee's decision.
+
+
+### Moderations of users and records
+
+![The moderation actions](./v12.0/user_moderation.png)
+
+The administration panel now includes a "User management" section to suspend, block and delete users, as well as undo all those actions.
+Suspending a user temporarily prevents them from creating records, but lets them appeal their suspension. Blocked users cannot sign in anymore and deleted users are removed from the system.
+
+Records can also be deleted (with a grace period for appeal or undoing) which empowers administrators to enforce institutional policies and fight spam.
+
+Bulk versions of these are in the works.
+
+### Even better previewers
+TODO
+
+### GitHub integration
+TODO
+
+
+### Changes from v12.0.0b1 are officialized
```

Comment:
do we see v12.0.0b1 as a official release? i mean on the same level as v11 and v12? i ask if we should incorporate the changes mentioned there into the v12 release note and not referee to v12.0.0b1 as a intermediate step between v11 and v12

Reply:
Do you mean we should erase the temporary version v12.0.0b1 completely from the docs and simply include its changes in these notes? I'd agree with that (less exceptions in docs always better). I'll do that.